test/fixed-point.jl

```diff
@@ -305,6 +305,16 @@ epsi{T}(::Type{T}) = eps(T)
     end
 end
 
+@testset "print" begin
+    @test sprint(print, FD2(1.00)) == "1.00"
```

Comment:
Originally the `show` tests were using `string` to check for `show` output and not `print`. I've updated this.